persistent-qq/test/PersistentTestModels.hs

```diff
@@ -110,11 +110,17 @@ share
     ~no Int
     def Int
 
+  PetAnimal schema=animals
+    ownerId PersonId
+    name Text
 |]
 
 deriving instance Show (BackendKey backend) => Show (PetGeneric backend)
 deriving instance Eq (BackendKey backend) => Eq (PetGeneric backend)
 
+deriving instance Show (BackendKey backend) => Show (PetAnimalGeneric backend)
+deriving instance Eq (BackendKey backend) => Eq (PetAnimalGeneric backend)
```

Comment:
Not sure why this is necessary. I get a weird error when I try to do this derivation with the standard `deriving Eq Show` syntax.